docs/pages/database-access/guides/mongodb-self-hosted.mdx

```diff
@@ -225,7 +225,7 @@ $ tsh db connect example-mongo
 ```
 
 <Admonition type="note" title="Note">
-  The `mongo` command-line client should be available in PATH in order to be
+  The `mongosh` command-line client should be available in PATH in order to be
```

Comment:
Should this mention `mongo` as a fallback?

Reply:
Good point, I'll mention it.